### ISSUE TITLE
Test params_filter with RSpec

### ISSUE THREAD

**Issue by nicolas-brousse** [palkan/action_policy#105]
I'm digging in testing policies those days and I've small questions.

I saw in documentation that there is an example for `:active_record_relation` scope. In our project we also use `.params_filter` who create a scope named `:action_controller_params`.

So my questions are, do you test `.params_filter` in your apps ? Do you think it's relevant ?  
And if it is, how you do it?

**Comment by palkan** [comment]
Hey!

> do you test .params_filter in your apps ? Do you think it's relevant ?
And if it is, how you do it?

TBH, I never did this with the actual Action Policy, only before I packed everything into a gem.
I think, it is relevant. You can do it like that:

```ruby
describe "params scope" do
  let(:user) { build_stubbed :user }
  let(:context) { {user: user} }

  let(:params) { {name: "a", password: "b"} }
  let(:target) { ActionController::Parameters.new(params) }

  # it's easier to asses the hash representation, not the AC::Params object
  subject { policy.apply_scope(target, type: :action_controller_params).to_h }

  context "as user" do
    it { is_expected.to eq({name: "a"}) }
  end

  context "as manager" do
    before { user.update!(role: :manager) }

    it { is_expected.to eq({name: "a", password: "b"})  }
  end
end
```

**Comment by nicolas-brousse** [comment]
@palkan thanks for your answer!

Looks good and interesting :)

Do you think we could add this example in the documentation?

**Comment by palkan** [comment]
> Do you think we could add this example in the documentation?

Yeah, that a good idea. Would you like to propose a PR?

**Comment by nicolas-brousse** [comment]
Cool. I'll try to found some time yes. I've to do https://github.com/palkan/action_policy/issues/88 too.

**Comment by palkan** [comment]
Closed by #107
